### PR TITLE
SUS-2525 | remove $wgEnableVignette config var

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-2525

See https://github.com/Wikia/config/pull/2230 for a config change

### CODE REVIEW

**Comment by macbre** [comment]
Failing unit tests are `NavigationTest`.

File: includes/wikia/GlobalFunctions.php [hunk 1]
```diff
@@ -104,22 +104,14 @@ function print_pre( $param, $return = 0 )
...
-	// This setting should be images.developerName.wikia-dev.com or perhaps "localhost"
-	// FIXME: This needs to be removed. It should be encapsulated in the URL generation.
-	$overrideServer = !empty( $wg->DevBoxImageServerOverride ) && !$wg->EnableVignette;
-	if ( $overrideServer ) {
-		$url = preg_replace( "/\\/\\/(.*?)wikia-dev\\.(pl|us|com)\\/(.*)/", "//{$wg->DevBoxImageServerOverride}/$2", $url );
-	}
-
 	wfDebug( __METHOD__ . ": requested url $url\n" );
 	if ( substr( strtolower( $url ), -4 ) != '.ogg' && isset( $wg->ImagesServers ) && is_int( $wg->ImagesServers ) ) {
 		if ( strlen( $url ) > 7 && substr( $url, 0, 7 ) == 'http://' ) {
```
Comment (WikiaTech) [inline includes/wikia/GlobalFunctions.php:117]: MAJOR Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [imageServing.class.php#L343](https://github.com/Wikia/app/blob/e9afdc2c528a6df3ecd03c3ea2a5c382b756eafc/extensions/wikia/ImageServing/imageServing.class.php#L343): Remove this unused private "getLegacyUrl" method. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1144)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**